src/slack_annotations/format.py

```diff
@@ -85,17 +85,28 @@ def _build_annotation_summary(annotation: dict[str, Any]) -> str:
 
 
 def _build_annotation_fields(annotation: dict[str, Any]) -> list[dict[str, Any]]:
+    quote = None
     try:
         quote = _get_quote(annotation)
     except Exception:  # pylint:disable=broad-exception-caught
-        quote = NONE_TEXT
-
-    return [
-        {"type": "mrkdwn", "text": "*Quote:*"},
-        {
-            "type": "mrkdwn",
-            "text": f"*Annotation* (<{annotation['links']['incontext']}|in-context link>):",
-        },
-        {"type": "plain_text", "text": quote},
-        {"type": "plain_text", "text": _get_text(annotation)},
-    ]
+        pass
+    incontext_link = annotation["links"]["incontext"]
+
+    if quote:
+        fields = [
+            {"type": "mrkdwn", "text": "*Quote:*"},
+            {
+                "type": "mrkdwn",
+                "text": f"*Annotation* (<{incontext_link}|in-context link>):",
+            },
+            {"type": "plain_text", "text": quote or NONE_TEXT},
+        ]
+    else:
+        fields = [
+            {
+                "type": "mrkdwn",
+                "text": f"*Page Note* (<{incontext_link}|in-context link>):",
```

Comment:
Don't particularly like how it's different now compared to annotation but we probably do want to save space and have both quote and annotation text be on the same row.
 
<img width="538" alt="Screenshot 2024-12-09 at 4 26 23 PM" src="https://github.com/user-attachments/assets/5479d207-058e-436d-bcaf-147aed3203fd">

Reply:
Looks good to me :+1: